### PR TITLE
rumqttc: example: basic client example transport Ws with mqttdashboar…

### LINKED ISSUE #343: rumqttc - add a websocket example
Request for action,

We don't have any example with rumqttc.

We should write a basic websocket client example.

@MathiasKoch, could you provides us an example?

### PR DESCRIPTION
**Pull request by IniterWorker**
Close #343 with a basic example.

### CODE REVIEW

**Comment by tekjar** [comment]
@de-sh Please verify this and merge

File: rumqttc/examples/websocket.rs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+use std::error::Error;
+use std::time::Duration;
+
+#[tokio::main(worker_threads = 1)]
+async fn main() -> Result<(), Box<dyn Error>> {
+    pretty_env_logger::init();
+
+    // port parameter is ignored when scheme is ws  
+    let mut mqttoptions = MqttOptions::new("clientId-aSziq39Bp3", "ws://broker.mqttdashboard.com:8000/mqtt", 8000);
+    mqttoptions.set_transport(Transport::Ws);
```
Comment (de-sh) [inline rumqttc/examples/websocket.rs:13]: How about adding a feature gated panic here to inform the user of the need to use `--features=websocket`?

**Review by de-sh — COMMENTED** [review]
Thanks for the PR. Please review my suggestion. I have opened a [PR on your fork](https://github.com/IniterWorker/rumqtt/pull/1) to solve the issue.

File: rumqttc/examples/websocket.rs (hunk 1 above)
Comment (de-sh) [reply rumqttc/examples/websocket.rs:13]: @IniterWorker?

**Review by de-sh — APPROVED** [review]
LGTM

**Comment by de-sh** [comment]
Thank you for your contribution @IniterWorker

**Pull request merged**